modules/EvseManager/IECStateMachine.cpp

```diff
@@ -182,7 +183,11 @@ std::queue<CPEvent> IECStateMachine::state_machine() {
             // Table A.6: Sequence 7 EV stops charging
             // Table A.6: Sequence 8.2 EV supply equipment
             // responds to EV opens S2 (w/o PWM)
-            connector_lock();
+            if (lock_connector_in_state_b) {
+                connector_lock();
+            } else {
+                connector_unlock();
```

Comment:
Add here the warning like `EVLOG_warning << "This violates IEC61851-1:2019 D.6.5 Table D.9 line 4 and should not be used in public environments!"`